src/shared/components/InActivityWarningModal/InActivityWarningModal.tsx

```diff
@@ -97,6 +98,17 @@ const InActivityWarningModal: FC<InActivityWarningModalProps> = ({
 			setIsTimedOut(true);
 			onForcedExit();
 		}
+
+		// AVO-2846: show timer before tab title when timer starts counting down
+		if (remainingTime < maxIdleTime) {
+			document.title =
+				format(new Date(remainingTime * 1000), 'mm:ss') + ' | ' + documentTitle;
```

Comment:
use this existing helper instead:
```suggestion
				formatDurationMinutesSeconds(remainingTime) + ' | ' + documentTitle;
```